campaignion_layout/src/Lookup.php

```diff
@@ -70,10 +70,12 @@ public function getTheme() {
    * @return string|null
    *   The name of the first enabled layout of the currently active theme.
    */
-  public function getLayout() {
-    foreach ($this->iterateItems() as $item) {
-      if (($theme = $this->themes->getTheme($item['theme'])) && $theme->isActive() && ($layout = $theme->getLayout($item['layout']))) {
-        return $layout;
+  public function getLayout(string $active_theme_name) {
```

Comment:
The new argument still needs a doc string ;)

Reply:
I’ve removed the parameter again by moving the logic into a new `Theme::getLayoutFromItems()`.